README.md

```diff
@@ -47,7 +47,8 @@ When the control panel is enabled:
 1. Select the **GeometryManager** GameObject in the Hierarchy window.
 2. Click on the **Load New Model** button in the Inspector window.
 3. Navigate to **"Assets/Models/"** and select one of the \*.gltf files.
```

Comment:
It is a bit confusing that the "Selected wall" in the model is called a certain name according to (presumably) the material that was given in the Sketchup model. This is confusing, because the Selected Material is not very obvious and does not grab that much attention.

Reply:
Noted!